packages/ramda-extension/src/mapOver.js

```diff
@@ -0,0 +1,28 @@
+import {uncurryN, map, over, lensProp} from 'ramda';
+
+/**
+ * Maps over a specific property on a list of objects, it returns a new list of objects
+ *
+ * @func
+ * @category List
+ *
+ * @param {Function} fn The function to be called on every specified property element of the `list`.
+ * @param {string} prop The property to be modified.
+ * @param {Array} list The list to be iterated over.
+ * @return {Array} The new list.
+ *
+ * @example
+ *
+ *		const objs = [
+ *			{ value: 1 },
+ *			{ value: 2 },
+ *			{ value: 3 },
+ *		];
+ *
+ *		R_.mapOver(R.add(100), "value", objs) // [{ value: 101 }, { value: 102 }, { value: 103 }]
+ *
+ * @sig (* -> *) -> string -> [object] -> [object]
+ */
+const mapOver = uncurryN(3, (fn, prop) => map(over(lensProp(prop), fn)));
```

Comment:
1. I think the idea of using `lens` to map is great. To make it work it would be better to move responsibility of creating lens outside to the user of the function.

2. the passing of `fn` and `lenses` should have the same order as the `order` function.

My first take: http://bit.do/fgkjF

addendum: We try use `[pointfree` style wherever possible](https://github.com/tommmyy/ramda-extension/blob/master/CONTRIBUTING.md). If then the function is too complex to read the whole utility function and its usefulness must be rethought again.

Reply:
My two centrs: in addition to this, including `mapOverProp` and `mapOverPath` would simplify usage even more.

Reply:
I have to admit that my level of point-free skills is still improving, although the paths that the variables have to follow isn't really linear in this case

Well, at this point why not creating the three version of the map over? 
`mapOver`: to use custom lenses
`mapOverProp`: to modify the specific prop (which is a functionality quite common for what I saw)
`mapOverPath`: to modify the specific path

And these three could be the different point-free implementations, following your version https://tinyurl.com/tdfzuyc

Reply:
I like it! Thank you :)

Reply:
My pleasure :D

I've rebased the fork with you last changes, modified the mapOver function and added the mapOverPath and mapOverLens alternatives, all tested and documented, hopefully I didn't miss anything